nautobot/extras/tests/test_forms.py

```diff
@@ -0,0 +1,323 @@
+from django.contrib.contenttypes.models import ContentType
+from django.test import TestCase
+
+from nautobot.dcim.forms import DeviceForm
+import nautobot.dcim.models as dcim
+from nautobot.ipam.forms import IPAddressForm, VLANGroupForm
+import nautobot.ipam.models as ipam
```

Comment:
Just to improve clarity can you please rename these to `dcim_models` and `ipam_models` respectively?